base/linalg/cholesky.jl

```diff
@@ -23,24 +47,27 @@ function CholeskyPivoted{T}(A::AbstractMatrix{T}, uplo::Char, piv::Vector{BlasIn
     CholeskyPivoted{T,typeof(A)}(A, uplo, piv, rank, tol, info)
 end
 
-function chol!{T<:BlasFloat}(A::StridedMatrix{T}, ::Type{UpperTriangular})
```

Comment:
So this renames the two-argument `chol!` method with the upper/lower type second argument, making it no longer available under the `chol!` name at all, right? I did a quick grep of packages and luckily it looks like no one was calling that in anything that's registered.